src/coreclr/jit/lsra.cpp

```diff
@@ -698,6 +710,12 @@ LinearScan::LinearScan(Compiler* theCompiler)
 
     availableFloatRegs  = RBM_ALLFLOAT;
     availableDoubleRegs = RBM_ALLDOUBLE;
+#if defined(TARGET_XARCH)
+    if (compiler->DoJitStressEvexEncoding())
+    {
+        availableMaskRegs = RBM_ALLMASK;
+    }
```

Comment:
Does this actually need to be conditioned? Presumably `availableMaskRegs` is completely unused if `DoJitStressEvexEncoding` is false so setting it as "available" wouldn't cause issues?

Reply:
It is, so yes, makes sense to just let it default.